requirements/docs.in

```diff
@@ -1,14 +1,13 @@
 -r base.in
-gitpython
+autodoc_pydantic
 ipykernel
-ipympl
 ipython!=8.7.0  # Breaks syntax highlighting in Jupyter code cells.
-ipywidgets
-matplotlib
+myst-parser
 nbsphinx
 plopp
+pydata-sphinx-theme>=0.13
 sphinx
-sphinx-autodoc-typehints==1.18.3  # later versions require sphinx >=5 which is incompatible with current sphinx-book-theme
-sphinx-book-theme==0.3.3
+sphinx-autodoc-typehints==1.23.0  # Higher versions require sphinx-7, which is not supported by sphinx-design yet
```

Comment:
cf your other PR, has this not been fixed now?

Reply:
Yes. But this was not in the template when I applied it here. I can wait with merging until the changes in the template are in.